cocos/2d/renderer/render-data.ts

```diff
@@ -23,11 +23,12 @@
 */
 
 import { DEBUG, JSB } from 'internal:constants';
+import { assert } from '@base/debug';
+import { assertIsTrue } from '@base/debug/internal';
```

Comment:
`assetIsTrue` is not public, I think it should be in internal export, they are not defferent modules, but different exprot condition

Reply:
Why can't it be public?

Reply:
I just wanna keep it internal, because we haven't export this interface ever before, 
And it's weird to provide this interface originally, because we have `assert` already, why we still provide `assertIsTrue`, I think it's a interface desigh issue.

Reply:
If so, we should not use it internally too. Though i know it is nothing about this issue.